spec/takeaway_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'takeaway'
+
```

Comment:
really good use of doubles and fully testing functionality.  Your code is clear and easy to understand.  Great job!